app/src/main/jni/enhance.h

```diff
@@ -1,6 +1,9 @@
 
 #include <opencv2/core/core.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
+#include <stdlib.h>
```

Comment:
Why do you need these libs?

Reply:
@abishekvashok I was trying a different method before which required stdlib.h

Reply:
Hm